app/models/movie.rb

```diff
@@ -1,3 +1,12 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_and_belongs_to_many :actors
```

Comment:
This is not the best association to set up for this many to many relationship because we need to have the joins table (actors_movies) be its own entity, while this association just references 2 different models. What you can use here instead is `has_ many :actors, through: :actors_movies`